### DISCUSSION TITLE
Why I can't use the copilot in VScode

### DISCUSSION THREAD

**Discussion by wangjiyuan9** [community/community#11497] [Copilot Conversations]
I have pass the wait list and installed the extensions had been installed, but I cant use it any more, there is nothing code hint.

**Comment by davecheney** [comment]
This error indicates that you are either not part of the beta invite list, or there was an error connecting to api.github.com to request a token to use the Copilot service. Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.
